client/components/ScenariosList/index.jsx

```diff
@@ -11,38 +12,92 @@ class ScenariosList extends Component {
     constructor(props) {
         super(props);
 
-        const category = this.props.location.pathname.slice(1);
+        const { category } = this.props;
 
         this.state = {
-            category
+            category,
+            listHeading: '',
+            scenariosToList: []
         };
+
+        this.sortDeletedScenarios = this.sortDeletedScenarios.bind(this);
+        this.getScenariosList = this.getScenariosList.bind(this);
     }
 
     async componentDidMount() {
         await this.props.getScenarios();
+        this.getScenariosList();
```

Comment:
`await this.getScenariosList();`

But also, I think this should be renamed to `reduceScenarios()`, since that's what it does.